test/suite/test_error_info.py

```diff
@@ -29,12 +29,14 @@
 import wiredtiger, wttest, time
 from wtdataset import SimpleDataSet
 
+WT_NONE = -32000
```

Comment:
In the folder `test/suite`, I can see we do use `wiredtiger.xyz` to access WiredTiger error codes, for instance: `wiredtiger.WT_NOTFOUND`. Could we do the same for this error code?

Reply:
Done in [Added assert to check for valid sub_level_err, formatting changes](https://github.com/wiredtiger/wiredtiger/pull/11350/commits/f0f443ad444cf6fe5f2fa02603569cd679c53dd6)